src/DatePicker/Utilities.elm

```diff
@@ -543,3 +532,24 @@ doDaysMatch zone dateTimeOne dateTimeTwo =
             Time.posixToParts zone dateTimeTwo
     in
     oneParts.day == twoParts.day && oneParts.month == twoParts.month && oneParts.year == twoParts.year
+
+
+calculatePad : Weekday -> Weekday -> Bool -> Int
+calculatePad firstWeekDay monthStartDay isFrontPad =
+    let
+        reversedListOfWeekDay =
+            generateListOfWeekDay firstWeekDay
+
+        calculatedPadInt =
+            case List.elemIndex monthStartDay reversedListOfWeekDay of
+                Just val ->
+                    if isFrontPad then
+                        -val
+
+                    else
+                        List.length reversedListOfWeekDay - val
```

Comment:
This would always equate to `7 - val` right? Might be a bit clearer just to put that unless there is a specific reason to reference the list length (over a hardcoded value).

Reply:
Yes I thought about that too. I'll change that, because I don't think we'll get a new weekday anytime soon...